### DISCUSSION TITLE
Can I use the token and validate meshcentral which is hosted in sub domain.

### DISCUSSION THREAD

**Discussion by amoljagdalepucsd** [Ylianst/MeshCentral#5184] [Q&A]
I have application which has api token and hosted on www.abc.com
now can I use that token to validate mesh central which is hosted on the www.abc.com/mehscetral.

Do you have any thoughts on this ?
@si458

**Comment by si458** [comment]
You can't run meshcentral from a sub folder it must be a subdomain

**Reply by si458** [reply]
I think you are misunderstanding 

`meshcentral.mydomain.com` would be a subdomain

`mydomain.com/meshcentral` is a folder! Not a subdomain!

Folders don't work!

You just use a subdomain!

**Comment by silversword411** [comment]
Is it just me, or is it bad Github etiquette to tag devs on everything on a first question. You should only do when continuing a conversation. They're already well aware of everything should they wish to know.

Also you should consider this: https://github.com/Ylianst/MeshCentral/discussions/4795

**Reply by si458** [reply]
i am only a community dev as well, not a main developer
